Update the readme and fix spelling (#1)
coblox/2019-genesis-workshop: README.md

Welcome to the workshop!

> The workshop was tested for Ubuntu and Mac only. If there are no instructions for your system then unfortunately there is nothing to be done.

Reviewer: ```suggestion
In the instructions below, if nothing is specified for your system (Mac/Ubuntu) then you skip the step.
```

# Slack channel

1. Go to #coblox-workshop

> 2. The url of the repo in the slack-channel topic, clone it: `git clone https://github.com/coblox/2019-genesis-workshop`

Reviewer: ```suggestion
2. The url of the repo in the Slack channel topic, clone it: `git clone https://github.com/coblox/2019-genesis-workshop`
```

   - `cargo install --path application/btsieve`

## Bitcoin setup
1. Navigate to the `2019-genesis-workshop` folder

> 2. In `docker-compose.yml`, replace `BITCOIN_NODE_IP_PORT` with the Bitcoin Node IP:PORT from the slack-channel topic 

Reviewer: ```suggestion
2. In `docker-compose.yml`, replace `BITCOIN_NODE_IP_PORT` with the Bitcoin Node IP:PORT from the Slack channel topic 
```

3. Start the Bitcoin node from the workshop folder: `docker-compose up`

## Ethereum setup
1. Install metamask: https://metamask.io/
2. Setup Metamask by following the wizard (new wallet). Yes, you have to do the seed phrase thing.
3. Connect metamask to our Ethereum node:
  - Choose "Custom RPC"
    
    ![connect Ethereum](./img/eth_connect.png)
    
  - Network Name: "CoBloX Test"

>   - New RPC URL: Ethereum URL from the slack-channel topic

Reviewer: ```suggestion
  - New RPC URL: Ethereum URL from the Slack channel topic
```

## Do a Swap!
For Ethereum actions: use metamask. Fund only ONCE.

For Bitcoin actions you need to use the commandline:

> - `alias btc-cli='docker exec -it bitcoind_1 bitcoin-cli -regtest -rpcport=18443 -rpcuser=bitcoin -rpcpassword=54pLR_f7-G6is32LP-7nbhzZSbJs_2zSATtZV_r05yg='`

Reviewer 2: This is sufficient.

```suggestion
- `alias btc-cli='docker exec bitcoind_1 bitcoin-cli -regtest'`
```
